jams/eval.py

```diff
@@ -54,10 +54,7 @@ def validate_annotation(ann, namespace):
         If `ann` fails schema validation
     '''
 
-    if not ann.search(namespace=namespace):
-        raise NamespaceError('Expected namespace="{:s}", found "{:s}"'
-                             .format(namespace, ann.namespace))
-
+    ann = convert(ann, namespace)
```

Comment:
The test was previously

"Is this annotation of namespace `X`?"

and is now

"can this annotation be interpreted as a valid instance of namespace `X`".


I think for evaluation (and sonification) the latter is what we really want.